mock/docs/release-instructions.txt

```diff
@@ -24,11 +24,25 @@ Release checklist overview:
    $ git pull --rebase main
 2) install snapshot version
    $ tito build --rpm -i
-3) run 'make check' in ./integration-tests directory and fix any reported
-   failures until it passes, you need to configure the box by ./setup-box.sh
-   first (don't run this on your box ideally, but temporarily allocated VM):
-   $ ./integration-tests/prepare/setup-box.sh
-   $ make check 1>/tmp/test-output.txt  2>&1
+3) run the testsuite, and fix any reported failures
+   It's strongly discouraged to run the testsuite locally, because certain
+   system configuration is needed.  Please allocate some disposable VM, and run
+   the testsuite there.  The system should be preconfigured using Ansible,
+   define the inventory (/etc/ansible/hosts):
```

Comment:
I prefer to have `ansible.cfg` in git root with:
```
[defaults]                                                                                                                                                                                   
inventory = ./inventory
```
And put this file in .gitignore

Reply:
This though limits the directories where to run the playbook from :-/ otoh if you edit the global inventory you can run the playbook from anywhere...

Reply:
OK, I will add a simple shell script, so users don't run ansible-playbook directly.